src/esp/metadata/attributes/SceneAttributes.cpp

```diff
@@ -102,8 +103,8 @@ std::string SceneAOInstanceAttributes::getSceneObjInstanceInfoInternal() const {
     initJointVels.append(std::to_string(it.second)).append(1, ',');
   }
   initJointVels.append("]");
-  return cfg.value("fixed_base").append(1, ',') + initJointPose.append(1, ',') +
-         initJointVels.append(1, ',');
+  return getAsString("fixed_base").append(1, ',') +
+         initJointPose.append(1, ',') + initJointVels.append(1, ',');
 }  // SceneAOInstanceAttributes::getSceneObjInstanceInfoInternal()
```

Comment:
Also

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.